scripts/quests/windurst/Making_Amens.lua

```diff
@@ -0,0 +1,112 @@
+-----------------------------------
+-- Making Amens!
+-----------------------------------
+-- !addquest 2 8
+-- Kuroido-Moido : !pos -111 -4 101 240
+-----------------------------------
+require('scripts/globals/interaction/quest')
+require('scripts/globals/npc_util')
+require('scripts/globals/quests')
+require('scripts/globals/titles')
+require('scripts/globals/zone')
+-----------------------------------
+
+local quest = Quest:new(xi.quest.log_id.WINDURST, xi.quest.id.windurst.MAKING_AMENS)
+
+quest.reward =
+{
+    fame     = 75,
+    fameArea = xi.quest.fame_area.WINDURST,
+    title    = xi.title.HAKKURU_RINKURUS_BENEFACTOR,
+    gil      = 6000,
+}
+
+quest.sections =
+{
+    {
+        check = function(player, status, vars)
+            return status == QUEST_AVAILABLE and
+                player:hasCompletedQuest(xi.quest.log_id.WINDURST, xi.quest.id.windurst.MAKING_AMENDS) and
+                player:getFameLevel(xi.quest.fame_area.WINDURST) >= 4 and
+                not player:needToZone()
+        end,
+
+        [xi.zone.PORT_WINDURST] =
+        {
+            ['Kuroido-Moido'] = quest:progressEvent(280),
+
+            onEventFinish =
+            {
+                [280] = function(player, csid, option, npc)
+                    quest:begin(player)
+                end,
+            },
+        },
+    },
+
+    {
+        check = function(player, status, vars)
+            return status == QUEST_ACCEPTED and not player:hasKeyItem(xi.ki.BROKEN_WAND)
+        end,
+
+        [xi.zone.GARLAIGE_CITADEL] =
+        {
+            onEventFinish =
+            {
+                -- Mashira
+                [11] = function(player, csid, option, npc)
+                    if option == 0 then
+                        npcUtil.giveKeyItem(player, xi.ki.BROKEN_WAND)
+                    end
+                end,
+            },
+        },
+
+        [xi.zone.PORT_WINDURST] =
+        {
+            ['Kuroido-Moido'] =
+            {
+                onTrigger =
+                {
+                    quest:event(283)
+                }
+            }
```

Comment:
```suggestion
            ['Kuroido-Moido'] = quest:event(283),
```

Reply:
Does this syntax only worth with progressQuest? For some reason it never triggers the way you suggested, which is why I had to put it like this.

Reply:
Oh really? It should work this way. Interesting.